app/src/main/java/nl/hogeschoolrotterdam/projectb/data/room/AppDatabase.java

```diff
@@ -12,8 +16,15 @@
  * Created by maartendegoede on 23/04/2019.
  * Copyright © 2019 Anass El Mahdaoui, Hicham El Marzgioui, Wesley de Man, Maarten de Goede all rights reserved.
  */
-@Database(entities = {Memory.class, Media.class, Image.class, Video.class}, version = 1)
+@Database(entities = {Memory.class, Media.class, Image.class, Video.class}, version = 2)
 @TypeConverters({Converters.class})
 public abstract class AppDatabase extends RoomDatabase {
     public abstract MemoryDao memoryDao();
+
+    public static final Migration MIGRATION_1_2 = new Migration(1, 2) {
+        @Override
+        public void migrate(@NonNull SupportSQLiteDatabase database) {
+            database.execSQL("ALTER TABLE Memory ADD COLUMN memoryTypeIconId INTEGER NOT NULL DEFAULT '" + R.drawable.ic_map_adefaultl + "'");
```

Comment:
It still says `ic_map_adefaultl` here instead of `ic_map_adefault`. Please fix that.